src/cartridges/int_adyen_overlay/cartridge/scripts/handleCustomObject.js

```diff
@@ -165,7 +165,7 @@ function handle(customObj) {
             isAdyen
           ) {
             refusedHpp = true;
-          } else if (order.status === Order.ORDER_STATUS_FAILED) {
+          } else if (order.status.value === Order.ORDER_STATUS_FAILED) {
```

Comment:
Should you do the same for line 251?

Reply:
We could, but there it will already be converted to a String, so it should also work that way

Reply:
Do we need to make it consistent over the entire class?

Reply:
Let's do that during Notification refactoring